ContactList.cs

```diff
@@ -0,0 +1,60 @@
+﻿using HwCreateGame;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+public class ContactList
+{
+    private List<Contact> contacts;
+
+    public ContactList()
+    {
+        contacts = new List<Contact>();
+    }
+
+    // Добавление контакта
+    public void AddContact(Contact contact)
+    {
+        contacts.Add(contact);
+    }
+
+    // Поиск контактов по параметру
+    public List<Contact> SearchContacts(string parameter)
+    {
+        return contacts.Where(c => c.Name.Contains(parameter) || c.Surname.Contains(parameter) || c.Phone.Contains(parameter)).ToList();
```

Comment:
What about part of the task that mentioned using some collection optimized for the search?